tests/integration/tests/test_util/config.py

```diff
@@ -188,3 +188,6 @@
 
 # SONOBUOY_VERSION is the version of sonobuoy to use for CNCF conformance tests.
 SONOBUOY_VERSION = os.getenv("TEST_SONOBUOY_VERSION") or "v0.57.3"
+
+# PERF_DEFAULT_ROUNDS is the number of rounds to run for performance tests.
+PERF_DEFAULT_ROUNDS = int(os.getenv("TEST_PERF_DEFAULT_ROUNDS") or 5)
```

Comment:
```suggestion
PERF_DEFAULT_ROUNDS = int(os.getenv("TEST_PERF_DEFAULT_ROUNDS") or 20)
```
Let's put 20 here, 5 would lead to insufficient results.

Reply:
I think I just remove this altogether and let the performance test define this explicitly.